packer/cache_test.go

```diff
@@ -1,52 +0,0 @@
-package packer
-
-import (
-	"os"
-	"path/filepath"
-	"testing"
-)
-
-func TestCachePath(t *testing.T) {
```

Comment:
Why removing the tests ?

Reply:
As mentioned above, I'm having issues with creating tests with each path being different depending on the OS. If you have any examples of how to do this I would be glad to update as I tried to update the tests in place but hit a wall making them work for without needing to know the OS.

Reply:
Because the behaviors differ between OS, you will have to create test that test things under unix and under windows. One way to fix this would be to check using `runtime.GOOS == "windows"`, may be some table tests could be generated depending on the os type ?

Reply:
Updated to make different test files depending on OS similar to the OS specific cache and config files.

Reply:
Super good ! Thanks.